packages/web3-eth-personal/types/index.d.ts

```diff
@@ -18,10 +18,23 @@
  * @date 2018
  */
 
-import { provider, RLPEncodedTransaction, TransactionConfig } from 'web3-core';
+import * as net from 'net';
+import { provider, RLPEncodedTransaction, TransactionConfig, BatchRequest, Providers, Extension } from 'web3-core';
 
 export class Personal {
-    constructor(provider: provider);
+    constructor(provider?: provider, net?: net.Socket);
+
+    readonly givenProvider: any;
+    static readonly givenProvider: any;
+    static readonly providers: Providers;
+    readonly currentProvider: provider;
```

Comment:
shouldnt this be static as well?

Reply:
The property ``currentProvider`` is an instance property. Only the defined properties [here](https://github.com/ethereum/web3.js/blob/1.x/packages/web3-core/src/index.js#L81) are defined as static class properties.

Reply:
... and the referenced ``addProviders`` function is also not called on every package.